### PR TITLE
번역 완료 2건 반영

### PR DESCRIPTION
**Pull request by jehyunlee**
두 글의 번역 완료에 대한 Pull Request 입니다.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,13 +13,14 @@
 
 | 완료 | 진행중 | 원제목 | 원링크 | 번역제목 | 번역링크 | 
 | -- | -- | ------- | ---- | ------- | ---- |
-| <ul><li>- [ ] </li> | <ul><li>- [X] </li> | Understanding Fastai's fit_one_cycle method | [Link](https://iconof.com/1cycle-learning-rate-policy/) | 미정 | [링크](https://github.com/jehyunlee/texts/blob/master/1cycle-learning-rate-policy/text.md)
+| <ul><li>- [ ] </li> | <ul><li>- [ ] </li> |  | [Link]() | 미정 | [링크]()
```
Comment (kkweon) [inline README.md]:
```suggestion
| complete  | [Link]() | 미정 | [링크]()
```

File: README.md [hunk 2]
```diff
@@ -13,13 +13,14 @@
 
 | 완료 | 진행중 | 원제목 | 원링크 | 번역제목 | 번역링크 |
```
Comment (kkweon) [inline README.md]:
```suggestion
| Status | 원제목 | 원링크 | 번역제목 | 번역링크 | 
```

File: README.md [hunk 3]
```diff
@@ -13,13 +13,14 @@
...
 | -- | -- | ------- | ---- | ------- | ---- |
-| <ul><li>- [ ] </li> | <ul><li>- [X] </li> | Understanding Fastai's fit_one_cycle method | [Link](https://iconof.com/1cycle-learning-rate-policy/) | 미정 | [링크](https://github.com/jehyunlee/texts/blob/master/1cycle-learning-rate-policy/text.md)
+| <ul><li>- [ ] </li> | <ul><li>- [ ] </li> |  | [Link]() | 미정 | [링크]()
 
 ## 머신러닝/딥러닝 일반
 
 | 완료 | 진행중 | 원제목 | 원링크 | 번역제목 | 번역링크 | 
 | -- | -- | ------- | ---- | ------- | ---- |
-| <ul><li>- [ ] </li> | <ul><li>- [ ] </li> | Estimating an Optimal Learning Rate For a Deep Neural Network  | [Link](https://towardsdatascience.com/estimating-optimal-learning-rate-for-a-deep-neural-network-ce32f2556ce0) | 미정 | [링크]()
+| <ul><li>- [X] </li> | <ul><li>- [ ] </li> | Understanding Fastai's fit_one_cycle method  | [Link](https://iconof.com/1cycle-learning-rate-policy/) | Fast.ai의 fit_one_cycle method 이해하기 | [링크](https://github.com/jehyunlee/texts/blob/master/1cycle-learning-rate-policy/text.md)|
```
Comment (kkweon) [inline README.md]:
```suggestion
| Complete | Understanding Fastai's fit_one_cycle method  | [Link](https://iconof.com/1cycle-learning-rate-policy/) | Fast.ai의 fit_one_cycle method 이해하기 | [링크](https://github.com/jehyunlee/texts/blob/master/1cycle-learning-rate-policy/text.md)|
```

File: README.md [hunk 4]
```diff
@@ -13,13 +13,14 @@
...
+| <ul><li>- [X] </li> | <ul><li>- [ ] </li> | Estimating an Optimal Learning Rate For a Deep Neural Network  | [Link](https://towardsdatascience.com/estimating-optimal-learning-rate-for-a-deep-neural-network-ce32f2556ce0) | 심층신경망에 맞는 최적 학습률 평가 | [링크](https://github.com/jehyunlee/texts/blob/master/estimating_an_optimal_learning_rate_for_a_deep_neural_network/text.md)
```
Comment (kkweon) [inline README.md]:
```suggestion
| Complete | Estimating an Optimal Learning Rate For a Deep Neural Network  | [Link](https://towardsdatascience.com/estimating-optimal-learning-rate-for-a-deep-neural-network-ce32f2556ce0) | 심층신경망에 맞는 최적 학습률 평가 | [링크](https://github.com/jehyunlee/texts/blob/master/estimating_an_optimal_learning_rate_for_a_deep_neural_network/text.md)
```

**Review by kkweon — CHANGES_REQUESTED** [review]
Sorry, I can't type korean.
But, I wrote in English as a reference.

Can we merge two columns into one and write status in Korean?

Currently, it's breaking the format on the webpage

cc: @deep-diver

**Comment by jehyunlee** [comment]
I gladly agree with @kkweon 's suggestion.
Since I'm not really good at GitHub, please let me know what am I supposed to do.
Is there any changes I need to make, please?

File: README.md [hunk 5]
```diff
@@ -11,15 +11,16 @@
 
 ## fast.ai 코스 관련
 
-| 완료 | 진행중 | 원제목 | 원링크 | 번역제목 | 번역링크 | 
+| Status | 원제목 | 원링크 | 번역제목 | 번역링크 | 
 | -- | -- | ------- | ---- | ------- | ---- |
```
Comment (kkweon) [inline README.md]:
```suggestion
| ---- | ------- | ---- | ------- | ---- |
```

I think you should update this as well.
You can preview your markdown.

File: README.md [hunk 6]
```diff
@@ -11,15 +11,16 @@
 
 ## fast.ai 코스 관련
 
-| 완료 | 진행중 | 원제목 | 원링크 | 번역제목 | 번역링크 | 
+| Status | 원제목 | 원링크 | 번역제목 | 번역링크 |
```
Comment (kkweon) [inline README.md]: Can you write in Korean for consistency?

File: README.md [hunk 7]
```diff
@@ -11,15 +11,16 @@
 
 ## fast.ai 코스 관련
 
-| 완료 | 진행중 | 원제목 | 원링크 | 번역제목 | 번역링크 | 
+| Status | 원제목 | 원링크 | 번역제목 | 번역링크 | 
 | -- | -- | ------- | ---- | ------- | ---- |
-| <ul><li>- [ ] </li> | <ul><li>- [X] </li> | Understanding Fastai's fit_one_cycle method | [Link](https://iconof.com/1cycle-learning-rate-policy/) | 미정 | [링크](https://github.com/jehyunlee/texts/blob/master/1cycle-learning-rate-policy/text.md)
+| complete  | [Link]() | 미정 | [링크]()
```
Comment (kkweon) [inline README.md]: Can you write in Korean?

File: README.md [hunk 8]
```diff
@@ -11,15 +11,16 @@
...
 
 ## 머신러닝/딥러닝 일반
 
 | 완료 | 진행중 | 원제목 | 원링크 | 번역제목 | 번역링크 | 
 | -- | -- | ------- | ---- | ------- | ---- |
-| <ul><li>- [ ] </li> | <ul><li>- [ ] </li> | Estimating an Optimal Learning Rate For a Deep Neural Network  | [Link](https://towardsdatascience.com/estimating-optimal-learning-rate-for-a-deep-neural-network-ce32f2556ce0) | 미정 | [링크]()
+| Complete | Understanding Fastai's fit_one_cycle method  | [Link](https://iconof.com/1cycle-learning-rate-policy/) | Fast.ai의 fit_one_cycle method 이해하기 | [링크](https://github.com/jehyunlee/texts/blob/master/1cycle-learning-rate-policy/text.md)|
```
Comment (kkweon) [inline README.md]: Can you write in Korean instead of  _Complete_ ?

File: README.md [hunk 9]
```diff
@@ -11,15 +11,16 @@
...
+| Complete | Estimating an Optimal Learning Rate For a Deep Neural Network  | [Link](https://towardsdatascience.com/estimating-optimal-learning-rate-for-a-deep-neural-network-ce32f2556ce0) | 심층신경망에 맞는 최적 학습률 평가 | [링크](https://github.com/jehyunlee/texts/blob/master/estimating_an_optimal_learning_rate_for_a_deep_neural_network/text.md)
```
Comment (kkweon) [inline README.md]: Can you write in Korean instead of _Complete_ ?

**Review by kkweon — CHANGES_REQUESTED** [review]
Thank you. It looks great!

File: README.md (hunk 9 above)
Comment (jehyunlee) [reply README.md]: I did it!

**Review by kkweon — APPROVED** [review]
Looks great!

**Pull request merged**